### DISCUSSION TITLE
“Sticky” console output for results

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#4121] [Enhancement Queue]
Shown here: https://bsky.app/profile/ariperkkio.dev/post/3laduidocxc2l

Keeps the result counts visible while the other logs pass by.

**Comment by nhoizey** [comment]
Nice indeed!

**Comment by Ryuno-Ki** [comment]
Might be related to https://github.com/11ty/eleventy/issues/3506
